### ISSUE TITLE
calicoctl doesn't work on GCI.

### ISSUE THREAD

**Issue by matthewdupre** [projectcalico/calico-containers#1183]
The current `calicoctl` uses pyinstaller in a mode that extracts files to /tmp.

Unfortunately on GCI, /tmp is noexec, so calicoctl doesn't work:

```
kubernetes-master bin # ./calicoctl status
./calicoctl: error while loading shared libraries: libz.so.1: failed to map segment from shared object: Operation not permitted
```

This isn't the only problem with pyinstaller: #1178.

**Comment by matthewdupre** [comment]
This will get fixed by the go version of calicoctl in a couple of months, but in the meantime if anyone is having problems because of this please post here so we can appropriately prioritise a temporary fix.

**Comment by matthewdupre** [comment]
Felix now uses single-directory mode.  The remaining problem is at https://github.com/projectcalico/calico-containers/blob/c1ab02263f83bb43cf72c1311b53970991e32c90/Makefile#L65.

**Comment by caseydavenport** [comment]
@matthewdupre Another option here is using the `calico/ctl` container.

**Comment by caseydavenport** [comment]
Should be fixed in v1.0.0-beta.
